lib/endpoints/volumes.js

```diff
@@ -442,7 +526,7 @@ function validateUpdateVolumeInput(req, res, next) {
         validationErrs.push(latestErr);
     }
 
-    for (paramName in req.params) {
+    for (paramName in req.query) {
```

Comment:
Should we remove `'id'` from `VALID_PARAM_NAMES` above then, since it would be in `req.params` but not in `req.query`?

And should we use this as an opportunity to actually validate `req.params.id`?